execution/engine/engine.go

```diff
@@ -96,13 +96,13 @@ func (ee *Engine) NotifyForkchoiceUpdate(
 		// We do not bubble the error up, since we want to handle it
 		// in the same way as the other cases.
 		ee.metrics.markForkchoiceUpdateSyncing(req.State, err)
-		return payloadID, nil, nil
```

Comment:
I don't see why we should return anything other than nil payload and latestValidaHash when it errs